src/main/java/bg/codexio/springframework/boot/configuration/DialectAutoConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package bg.codexio.springframework.boot.configuration;
+
+import bg.codexio.springframework.data.jpa.requery.dialect.RequeryEnhancedMySQLDialect;
+import bg.codexio.springframework.data.jpa.requery.dialect.RequeryEnhancedPostgreSQLDialect;
+import org.hibernate.dialect.Dialect;
+import org.hibernate.dialect.OracleDialect;
+import org.hibernate.dialect.SQLServerDialect;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+@Configuration
+public class DialectAutoConfiguration {
```

Comment:
This shouldn't be here. Either people use this starter and the other starter in conjunction, or they provide dialect configuration themselves.

Reply:
removed